README.md

```diff
@@ -6,6 +6,8 @@
 
 ## Getting started
 
+We are no longer actively developing features for this app. PRs will be accepted for bug fixes, translation, and content updates. Active feature development is happening at [https://github.com/zooniverse/front-end-monorepo/](https://github.com/zooniverse/front-end-monorepo/)
```

Comment:
_Very_ minor, but translation(s) is singular here and plural in the `CONTRIBUTING` doc